docs/install/osx_setup.md

```diff
@@ -77,6 +77,10 @@ Install the dependencies, required for MXNet, with the following commands:
 
 ```bash
 	brew update
+    brew install python3
```

Comment:
Ah the implications of this. Let's do it, but I think we need to socialize how many tutorials are going to break and how many other things need to change.

Reply:
I haven't encountered any problems. Are the tutorials in CI? I think compat issues between py2 / py3 are very easy to diagnose by users on their own.

Reply:
I also prefer just use python in the instructon. Also, if user use virtualenv, this will not work.

Reply:
Should this be python instead of python3? Also it seems the community has not reached an agreement on retiring python2 sypport yet.